include/spdk/bdev.h

```diff
@@ -133,6 +133,7 @@ enum spdk_bdev_io_type {
 	SPDK_BDEV_IO_TYPE_UNMAP,
 	SPDK_BDEV_IO_TYPE_FLUSH,
 	SPDK_BDEV_IO_TYPE_RESET,
+	SPDK_BDEV_IO_TYPE_PASSTHRU,
```

Comment:
Just define two values in this enum instead of SPDK_BDEV_IO_TYPE_PASSTHRU:

SPDK_BDEV_IO_TYPE_NVME_ADMIN
SPDK_BDEV_IO_TYPE_NVME_IO